tech.json

```diff
@@ -866,6 +866,14 @@
             }
           ]
         },
+        {
+          "name": "canSlowShortCharge",
+          "requires": ["SpeedBooster"],
+          "note": [
+            "The ability to control Samus' run speed while shortcharging.  Specifically by releasing run after achieving blue.",
+            "Being able to 4 tap is expected as that brings out the most value from this tech."
```

Comment:
I dont have time to review this PR until Monday, but I was wondering if this 4 tap includes stutters? if not, could it be done with a stutter 3-tap (or 2?) and should that be mentioned?

Reply:
Stutters reduce the amount of runway used but doesn't change run speed.  Unless its a 0 tap but that will be treated as full run speed.  And acceptable runway length is entered on seed gen with `Shinespark tiles`.

A big assumption here is that if you can `slowShortcharge`, you can also short hop mockball.

The slow shortcharge branches were tested, but the longer runway branches are guesses.

And there could be an in between option that is ~2tap with short hop that wouldn't need a huge runway.
So defining short hop somehow would be nice.